Unity-Technologies/ScriptableRenderPipeline
# Bugfix for transparent objects and shadow receiving

### Purpose of this PR
Fixing issue #1116936
https://issuetracker.unity3d.com/issues/lwrp-cascaded-shadows-do-not-appear-on-alpha-blended-objects

---
### Testing status

**Manual Tests**: What did you do?
- [X ] Opened test project + Run graphic tests locally
- [X ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
Added a automatic test with a reference image.

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/universal%252Fbugfix-1116936
Any test projects to go with this to help reviewers?

The "105_TransparentReceiveShadows" scene in the UniversalGraphicsTest project has spheres using unlit, lit, simple lit, particles lit, particles simple lit and shadergraph shaders to easily see if things are working or not.

---
### Comments to reviewers
Transparent objects were not receiving shadows when using shadow cascades.

Things done:
1. Fixed the problem so now transparent objects should receive shadows with cascades both enabled and disabled.
2. Added a toggle "Transparent Receive Shadows" in the renderer asset which enables/disables globally that transparent object can receive shadows. Default setting is set to true.
3. Particles are now able to receive shadows. It was disabled because it didn't work with screen space shadows but since that is no longer used we were able to reenable the feature.
